packs/equipment-effects/effect-iron-medallion.json

```diff
@@ -0,0 +1,46 @@
+{
+    "_id": "xV9rME2l1f10mnF9",
+    "img": "systems/pf2e/icons/equipment/consumables/talismans/iron-medallion.webp",
+    "name": "Effect: Iron Medallion",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Iron Medallion]</p>\n<p>You gain a +2 status bonus to the triggering save and other saves against fear.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 10
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder GM Core"
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "fear"
+                ],
+                "selector": "saving-throw",
```

Comment:
The triggering save is specified to be Will, but it does not specify that subsequent saves must be.